public/index.html

```diff
@@ -7,7 +7,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Web site created using create-react-app" />
-  <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+  <link name="icon" rel="icon" href="%PUBLIC_URL%/movie-browser-icon.png" />
+  <link rel="apple-touch-icon" href="%PUBLIC_URL%/movie-browser-icon.png" />
   <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
```

Comment:
how do you like the new icon?

it will be the base for the whole color scheme and og image and theme so i need feedback

Reply:
i think this icon is perfect